torchinductor/codegen/wrapper.py

```diff
@@ -97,6 +98,10 @@ def __init__(self):
             ["", "", f"def call({', '.join(V.graph.graph_inputs.keys())}):"]
         )
         with self.prefix.indent():
+            for name in V.graph.randomness_seeds:
+                self.prefix.writeline(
+                    f"torch.randint(2**31, size=(), dtype=torch.int32, out={name})"
```

Comment:
my head spins thinking about which random counters are advanced here and how cuda graphs are handling it, but looks like it should work (at the cost of one extra kernel which in future we should be able to avoid).